server/data/apps.json

```diff
@@ -3777,5 +3777,35 @@
         "elementURL": "https://www.cooler.fm"
       }
     ]
-  }  
+  }, {
+    "appName": "Alitu",
+    "appType": [
+      "hosting",
+      "website",
+      "all-in-one podcast maker"
```

Comment:
I don't think that all-in-one is a valid appType
https://github.com/Podcastindex-org/web-ui/blob/master/server/data/apps-schema.json#L29-L44